lib/aws_auditor/ec2_instance.rb

```diff
@@ -54,8 +66,8 @@ def platform_helper(ec2_instance)
         else
           return 'Linux'
         end
-      elsif ec2_instance.class.to_s == 'AWS::EC2::ReservedInstances'
```

Comment:
The API doesn't explicitly say this is a thing, but it looks like it is. I'll put it back in.